src/map_server/include/map_server/base_map_loader.h

```diff
@@ -0,0 +1,37 @@
+#ifndef MAP_SERVER_BASE_MAP_LOADER_H
```

Comment:
Please run cpplint and uncrustify on all of the source files (a requirement for integration of a PR).